lib/todo-list.js

```diff
@@ -0,0 +1,80 @@
+(function (factory) {
+    if (typeof define === 'function' && define['amd']) {
+        define(['knockout'], factory);
+    } else if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+        module.exports = factory(require('knockout'));
+    } else {
+        window.ToDoList = factory(ko);
+    }
+} (function (ko) {
+    return function ToDoList() {
+        'use strict';
+        let tasks = ko.observableArray([]);
+        let newTaskText = ko.observable("");
+
+        function addTask() {
+            if (newTaskText() === "") return;
```

Comment:
I've noticed this line of code is new. It makes sense that when you click the Add Task button, an empty task should not be added. But I think we should add a unit test for this. Can you do that please?